release/release_tests.yaml

```diff
@@ -1010,6 +1010,47 @@
 
   alert: default
 
+#######################
+# Lightning tests
+#######################
+
+- name: lightning_trainer_test_latest
+  group: Lightning tests
+  working_dir: lightning_tests
+
+  frequency: nightly-3x
+  team: ml
+
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: compute_tpl.yaml
+
+  run:
+    timeout: 1200
+    script: python workloads/test_trainer.py
+    wait_for_nodes:
+      num_nodes: 3
+
+  alert: default
```

Comment:
What does this do? 😮  Should it be a special "lightning" label like some xgboost tests do?

Reply:
hmm, good question. 80% release tests use default. Let me check out the meanings of these keywords.